### DISCUSSION TITLE
Conflicting device emulation parameters

### DISCUSSION THREAD

**Discussion by jlauro-sre** [GoogleChrome/lighthouse#12726] [Q&A]
Hi all,
Issue trying to force the desktop form factor. I've reviewed a few of the other posts similar to this and they didn't return the desktop form for me.

I am running lighthouse via npx. I've tried a modified config.js and option flags.

`npx lighthouse https://www.1800flowers.com/ --view --form-factor=desktop --screenEmulation.disabled --emulatedUserAgent="Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91"`

```
'use strict';
const constants = require('./constants.js');

module.exports = {
  extends: 'lighthouse:default',
  plugins: ['lighthouse-plugin-field-performance'],
  settings: {
        formFactor: 'desktop',
        psiToken: 'xxx',
  },
    screenEmulation: {
      mobile: false,
      width: 1600,
      height: 900,
      deviceScaleFactor: 1,
      disabled: false
    },
    emulatedUserAgent: 'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/84.0.4143.7 Safari/537.36 Chrome-Lighthouse'
}
```

Everything returns a mobile device type. What am I missing? I imagine a config mapping is missing somewhere.

**Comment by adamraine** [comment]
`formFactor` determines how the audits are scored but this does not determine how lighthouse loads the page. 

I think you are looking for the `--preset=desktop` flag. For reference, this is the config used by `--preset=desktop`:
https://github.com/GoogleChrome/lighthouse/blob/master/lighthouse-core/config/desktop-config.js

It looks like you tried to replicate this config, but your `screenEmulation` and `emulatedUserAgent` are outside the `settings` config.

**Reply by jlauro-sre** [reply]
Thank you for your reply.
I tried this flag and it does not seem to be a valid option.

```
[root@hoth lighthouse-plugin]# npx lighthouse "https://www.1800flowers.com/" --chrome-flags="--no-sandbox --headless" --preset=desktop
Invalid values:
  Argument: preset, Given: "desktop", Choices: "perf", "experimental"

Specify --help for available options
```

**Reply by connorjclark** [reply]
Try updating your version of Lighthouse, those were the preset options available in an older version.
